example/jaeger/main.go

```diff
@@ -21,26 +21,15 @@ import (
 	"log"
 
 	"go.opentelemetry.io/otel"
-	"go.opentelemetry.io/otel/sdk/resource"
-	"go.opentelemetry.io/otel/semconv"
 
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/exporters/trace/jaeger"
-	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 )
 
 func main() {
 	// Create and install Jaeger export pipeline as the global.
 	tp, err := jaeger.InstallNewPipeline(
 		jaeger.WithCollectorEndpoint("http://localhost:14268/api/traces"),
-		jaeger.WithSDKOptions(
-			sdktrace.WithSampler(sdktrace.AlwaysSample()),
-			sdktrace.WithResource(resource.NewWithAttributes(
-				semconv.ServiceNameKey.String("trace-demo"),
```

Comment:
If a service name isn't set here, what service name is used?  The one from the default resource?  Should we use `NewRawExporter` here instead, or maybe have an option for adding a service name through this function?

Reply:
Yeah, it will be the service name from the default resource. I'm okay with switching this to use the `NewRawExporter` and show how to set the default service name that way.

If we want to updated this to accept a service name we should probably do it for all the exporters. That work seems like it should be a separate PR/Issue if we want to go that direction.

Reply:
Updated. @Aneurysm9 PTAL

Reply:
LGTM